crates/sui-core/src/consensus_adapter.rs

```diff
@@ -459,36 +474,65 @@ impl ConsensusAdapter {
     fn check_submission_wrt_connectivity_and_scores(
         &self,
         positions: Vec<AuthorityName>,
-    ) -> (usize, bool, AuthorityName) {
+    ) -> (usize, usize, String, AuthorityName) {
         let low_scoring_authorities = self.low_scoring_authorities.load().load_full();
         let first_submitter = *positions.first().unwrap_or(&AuthorityName::default());
         if low_scoring_authorities.get(&self.authority).is_some() {
-            return (positions.len(), true, first_submitter);
+            return (
+                positions.len(),
+                positions.len(),
```

Comment:
We should say zero for `positions_moved` in this case, just want to track the case below when validators have been filtered and the validator moved up in the queue.

Reply:
this case will never result in a metric being emitted, but sure